cost/kubecost/sizing/README.md

```diff
@@ -13,6 +13,7 @@ Kubecost request sizing recommendations policy provides Kubecost recommendations
 
 - *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
 - *Kubecost Host or IP address* - The host or IP Address where your Kubecost runs
+- *Recommendations request method* - The method used to request savings recommendations: "overall" or per "cluster"
```

Comment:
This should be renamed (both here and in the policy) to something more specific to the functionality. Other policies refer to this as "Scope", which would be a good name for this parameter.